lib/src/dart2_suggestors/props_and_state_companion_class_remover.dart

```diff
@@ -24,8 +24,8 @@ import '../util.dart';
 class PropsAndStateCompanionClassRemover extends RecursiveAstVisitor
     with AstVisitingSuggestorMixin
     implements Suggestor {
-  bool shouldRemoveCompanionClassFor(
-          ClassDeclaration candidate, CompilationUnit node) =>
+  bool shouldRemoveCompanionClassFor(ClassDeclaration candidate,
+          ClassDeclaration companion, CompilationUnit node) =>
       true;
```

Comment:
I added this because there was an existing test for `'when the stubbed "companion" class(es) are publicly exported'` and so it seemed like it should be checking `companion.isPublic()` instead of `candidate.isPublic()`. But it looks like Aaron changed it to `ClassToMixinConverter.convertedClasses` like you said which makes it a lot simpler so I'll revert my changes here.

Reply:
Actually, I can't remove the `SemverHelper` usage in this suggestor right now because `shouldMigrateSimplePropsAndStateClass` and `shouldMigrateAdvancedPropsAndStateClass` need the `SemverHelper` argument. I put notes to remove this once Aaron's changes that remove those two utility function usages are merged in.